### ISSUE TITLE
[android.ipwebcam] Device onbereikbaar

### ISSUE THREAD

**Issue by Povlov** [athombv/homey#2113]
**Metadata**

* **Homey Version**: Homey Firmware Versie: 1.5.3
* **Homey Log Code**:
* **Smartphone Platform**: android
* **Smartphone App Version**: 1.13.24

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

sustain the connection of the device

**What did you expect as result?**

the device continuously connected

**What was the result?**

after a period of x seconds, equal to the setting in "polling", the broken link symbol appears, together with "onbereikbaar" box on hovering mouse.

**Any other remarks**
Tried many settings in phoneapp.
Bottom line ¨ Video Connection:0 " shows a "1" only first time "test connection", when the phoneapp "start server" is fresly started, and then a new device is created on homey.  
![screenshot 2017-12-28 at 22 13 20](https://user-images.githubusercontent.com/24312420/34423662-d505c198-ec1d-11e7-9a5a-e3010caff961.png)

**Comment by bvdbos** [comment]
This is probably an issue with the app?
https://github.com/jghaanstra/android.ipwebcam

**Comment by jghaanstra** [comment]
This ended up in the wrong repo due to the mentioned issue above.

About the actual issue, I have submitted a bugfix to the app store 2 days ago that potentially fixes this issue. This issue manifests itself with Android devices that do not have a lux sensor. Perhaps this is also the same with you. Please await this new release to see if the problem is solved.

**Comment by Povlov** [comment]
Tx!

**Comment by jeroenvollenbrock** [comment]
I'm closing this issue as it is supposedly fixed in the latest update. If the problem persists, please let me know.
